service/api/serializers.py

```diff
@@ -5,6 +5,6 @@
 class WatchSerializer(serializers.ModelSerializer):
     class Meta:
         model = WatchList
-        fields = '__all__'
+        fields = ['id', 'title', 'poster', 'description', 'tmdBid_id', 'rating', 'watch_list']
```

Comment:
Typo: `tmdBid_id` should be renamed to `tmdbId` (https://github.com/WaveOfCoding/watchtower/blob/main/service/spec/http/api.yaml#L70)

`watch_list` should be renamed to `watchlist` (https://github.com/WaveOfCoding/watchtower/blob/main/service/spec/http/api.yaml#L74)